htdecodetoken

```diff
@@ -11,11 +11,14 @@ usage()
     echo 'Decodes a JSON Web Token'
     echo '  -a: show algorithm portion of JWT'
     echo '  -H: show dates in human readable format instead of epoch'
+    echo '  -s: skip scitokens-verify validation (cannot be used with -f).'
```

Comment:
One more nit: please remove the trailing period on this line.

Reply:
Done.  Did you also want the period removed two lines later, after "PATH" at the end of that line? I didn't want to have a big run-on sentence, but stylistically, I do see that none of the other flag descriptions have periods, so it's up to you.

Reply:
I think it's OK because the other notes at the end have periods.